examples/simple-nightwatch/SimpleNightwatch.elm

```diff
@@ -53,15 +58,30 @@ update msg ({ date, datePicker } as model) =
 
                 newDate =
                     case dateEvent of
-                        Changed changedDate ->
-                            changedDate
+                        Picked changedDate ->
+                            Just changedDate
 
                         _ ->
                             date
+
+                error =
```

Comment:
I'd have the error and new day being set inside the same `case`. Something like:

```
(newDate, newError) = 
   case dateEvent of
      ....
```

Might be it's personal preference, but there's just one POT there to set both variables at a same time.